index.html

```diff
@@ -372,12 +375,20 @@ <h3>Use Cases and Requirements</h3>
 <a>Holders</a> control which <a>verifiable credentials</a> to use and when.
         </li>
         <li>
-<a>Holders</a> are able to freely choose and change the agents they employ
-to help them manage and share their <a>verifiable credentials</a>.
+<a>Holders</a> are be able to freely choose and change the agents they employ
+to help them manage their <a>verifiable credentials</a> and generate and share <a>verifiable presentations</a>.
+        </li>
+        <li>
+<a>Holders</a> that share <a>verifiable presentations</a> that can be verified without revealing
+the identity of the <a>verifier</a> to the <a>issuer</a>. Even though this specification does not
+disallow such behavior, it strongly discourages for the holder to preserve its privacy.
```

Comment:
Ok, i can reword the sentence, but should i move the discouragement part to the Privacy section

Reply:
Yeah, I'd move any privacy-related recommendations to the privacy section. It may be a good idea to just strike it from this PR and add it to another PR that's dealing with that section -- as I presume there's more to say there.

Reply:
The PR by @dhh1128  containing the definition of predicate has been already approved and merged so i am removing my definition

Reply:
Removed